compiler/testData/codegen/box/reflection/call/inlineClasses/simpleConstructorWithInlineClassParameter.kt

```diff
@@ -0,0 +1,18 @@
+// IGNORE_BACKEND: JS_IR
```

Comment:
I added a test case in reference to `testData/codegen/box/reflection/call/simpleConstructor.kt`, but I couldn't come up with a good name.
Also, I am not sure about the `IGNORE` setting.

Reply:
Please use `// TARGET_BACKEND: JVM` instead of the IGNORE_BACKEND directives

Reply:
Thank you, the following commit fixes.
[6985903](https://github.com/JetBrains/kotlin/pull/5093/commits/69859039865e8f7c631d5a6bc682a8b793c19a02)